sunpy/map/sources/tests/test_trace_source.py

```diff
@@ -0,0 +1,35 @@
+import os
+import glob
+
+import pytest
+
+from sunpy.map.sources.trace import TRACEMap
+from sunpy.map import Map
+import sunpy.data.test
+
+path = sunpy.data.test.rootdir
+fitspath = glob.glob(os.path.join(path, "tsi20010130_025823_a2.fits"))
```

Comment:
Why do we need to use `glob` in here?

Reply:
No . I think I forgot to remove it , while experimenting with the code .